src/util/mpsc.rs

```diff
@@ -93,9 +112,11 @@ impl<T> Sender<T> {
     pub fn try_send(&self, t: T) -> Result<(), TrySendError<T>> {
         if !self.state.is_receiver_closed() {
             crossbeam_channel::select! {
-                send(self.sender, t) => Ok(()),
-                default => Err(TrySendError::Full(t)),
+                send(self.sender, t) => {},
```

Comment:
Is it equal to `self.sender.send(t)`?

Reply:
Why not put `notify` into the send operation body

Reply:
Reduce indent.

Reply:
But it can save an extra notification.

Reply:
I don't get it. What's the extra notification?

Reply:
Okay, misread. I thought `Full(t)` also causes a notification.